pkg/imageserver/webdav.go

```diff
@@ -69,6 +69,9 @@ func (s *webdavImageServer) GetHandler(meta *iiapi.InspectorMetadata,
 		if err := syscall.Chroot(ImageServeURL); err != nil {
 			return nil, fmt.Errorf("Unable to chroot into %s: %v\n", ImageServeURL, err)
 		}
+		if err := syscall.Chdir("/"); err != nil {
```

Comment:
Because `imageserver` does not imports `os` and the difference between `syscall.Chdir` and `os.Chdir` is that the `os.Chdir` wraps error. So by doing this I want to avoid unnecessary imports.

https://golang.org/src/os/file.go?s=6685:6713#L212